swan-accpy/Dockerfile

```diff
@@ -35,4 +35,8 @@ RUN pip install --no-deps --no-cache-dir \
     swancustomenvironments==0.0.12
 RUN echo -e '\nc.ServerApp.extra_template_paths = ["/opt/conda/lib/python3.11/site-packages/swancustomenvironments/templates/"]' >> /home/${NB_USER}/.jupyter/jupyter_server_config.py
 
-USER ${NB_UID}
\ No newline at end of file
+USER ${NB_UID}
+
+# Install UV package manager for Python (faster that pip)
+RUN curl -LsSf https://astral.sh/uv/install.sh | HOME=/home/${NB_USER} sh -s -- --no-modify-path && \
+    chmod +x /home/${NB_USER}/.local/bin/uv
```

Comment:
+x is u+x?

Reply:
Also, can uv be installed in a system directory that is findable by default in the path? So that we do not need to tweak path afterwards.